example/node/index.js

```diff
@@ -185,6 +187,25 @@ async function run() {
   const fs = require('fs');
   const imageBlob = await layerS2L2A.getMap(getMapParams, ApiType.WMS);
   fs.writeFileSync('./image.jpeg', imageBlob, { encoding: null });
+
+  // Cancel request example
+  // Create source and pass token as config on the request (getMap WMS/Processing and getStats supported for layerV3)
+  const source = CancelFactory.createSource();
+  setTimeout(() => {
+    printOut('Cancelling request');
+    source.cancel();
+  }, 100);
+  try {
+    const response = await layerS2L2A.getMap(getMapParams, ApiType.WMS, { cancelToken: source.getToken() });
+    // const dates = await layer.findDatesUTC(bbox, fromTime, toTime, requestConfig);
+    // const stats = await layer.getStats(getStatsParams, requestConfig);
```

Comment:
Is this something that should be cleaned up?

Reply:
The idea was to put examples on the different methods that can be cancelled and a visual example on how to request configuration was passed on. I agree it's not the cleanest thing, and maybe it's better to just leave the getMap on it.

Reply:
Not necessarily, you can also just uncomment them... IIUC the example works this way too? (of course, `requestConfig` needs to be defined or replaced)